README.md

```diff
@@ -50,17 +50,74 @@ engine.push(function(req, res, next, end){
 })
 ```
 
-RpcEngines can be nested by converting them to middleware `asMiddleware(engine)`
+RpcEngines can be nested by converting them to middleware using `asMiddleware(engine)`:
 
 ```js
-const asMiddleware = require('json-rpc-engine/lib/asMiddleware')
+const asMiddleware = require('json-rpc-engine/src/asMiddleware')
 
 let engine = new RpcEngine()
 let subengine = new RpcEngine()
 engine.push(asMiddleware(subengine))
 ```
 
-### gotchas
+### Asynchronous Middlewares
```

Comment:
The terminology in this section is a but confused, as _all_ of the middleware is asynchronous technically, because it's callback-based.

Reply:
I believe I cleared it up in: 2561ecf

Reply:
Ah, good idea to use `async`, that's much better